cli/.graphqlconfig

```diff
@@ -0,0 +1,16 @@
+{

```

Comment:
Is this used instead of a build.rs script?

Reply:
Oh sorry, I should have clarified. This is specifically for IntelliJ's graphql plugin integration. It allows for code highlighting when writing queries and some other features.